crates/orderbook/src/fee.rs

```diff
@@ -837,7 +839,7 @@ mod tests {
             ..Default::default()
         };
         let native_token_price_estimation_amount = 100.;
-        let sell_token_price = 1.25;
+        let sell_token_price = 0.1;
```

Comment:
Not an actually needed change - but if you make the sell token less valuable than the native token then it avoid rounding errors in the asserts 👇.